src/overview/components/FilterContainer.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import debounce from 'lodash/fp/debounce'
+import noop from 'lodash/fp/noop'
+
+import { remoteFunction } from 'src/util/webextensionRPC'
+import { Tags, TagRow } from 'src/common-ui/components'
+
+class FilterContainer extends Component {
```

Comment:
This container shares lots of state and logic code with `TagsContainer`. It would be best to either make an abstract (no `render()` method) `SelectionDropdown` (or whatever name) container class with all the logic and `extend` from that in `TagsContainer` and `FilterContainer`, or just a generic container that accepts the differences as props. Again, not urgent.